utils/test_move_and_process.py

```diff
@@ -40,6 +41,17 @@ class ProcessGenerate(unittest.TestCase):
     def setUp(self):
         logging.getLogger('').setLevel(logging.WARNING)
 
+        # Generate test audio
+        self.audiofile="sine.wav"
+        cmd = ('sox', '-b', '16',  '-n', self.audiofile,
+               'synth', '3', 'sine', '300-3300')
```

Comment:
I'm not thrilled with depending on `sox` and `convert` suddenly. It makes setting up harder. If we can find some tiny resources we can add to the repo, that'd be better. If bigger, (like video) we could have a build/setup-step that actually downloads the file locally (from frikanalen).

Reply:
I will add test files, but do not really understand why the fairly universally available programs sox and convert make setting up much harder.